Assets/Scripts/UI/UtilityBuildMenu.cs

```diff
@@ -0,0 +1,101 @@
+#region License
```

Comment:
Are we still using this Build files? I believe that the ConstructionMenu replaced them. We still have the Furniture one, which might need to be removed too.